test-clients/src/main/java/com/hedera/services/bdd/suites/perf/AccountBalancesClientSaveLoadTest.java

```diff
@@ -0,0 +1,315 @@
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKENS_PER_ACCOUNT_LIMIT_EXCEEDED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKEN_ALREADY_ASSOCIATED_TO_ACCOUNT;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKEN_NOT_ASSOCIATED_TO_ACCOUNT;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TRANSACTION_EXPIRED;
+
+
+public class AccountBalancesClientSaveLoadTest extends LoadTest  {
+	private static final Logger log = LogManager.getLogger(AccountBalancesClientSaveLoadTest.class);
+	final static int MAX_PENDING_OPS_FOR_SETUP = 10_000;
+	final static int TOTAL_ACCOUNT = 200_000;
+	final static int ESTIMATED_TOKEN_CREATION_RATE = 50;
+	final static int ESTIMATED_CRYPTO_CREATION_RATE = 500;
+	final static long MIN_ACCOUNT_BALANCE = 1_000_000_000L;
+	final static int MIN_TOKEN_SUPPLY = 1000;
+	final static int MAX_TOKEN_SUPPLY = 1_000_000;
+	final static int MAX_TOKEN_TRANSFER = 100;
+	final static int SECOND = 1000;
+	private static final Random r = new Random();
+
+	private final static int TOTAL_TEST_TOKENS = 500;
+	private final static String ACCT_NAME_PREFIX = "acct-";
+	private final static String TOKEN_NAME_PREFIX = "token-";
+
+	private static int normal_acct_created = 0;
+
+	private final static String ACCOUNT_FILE_EXPORT_DIR = "src/main/resource/accountBalancesClient.pb";
+
+	private int totalTestTokens = TOTAL_TEST_TOKENS;
+	private int totalAccounts = TOTAL_ACCOUNT;
+	private int totalTreasureAccounts = totalTestTokens;
+
+	List<Pair<Integer, Integer>> tokenAcctAssociations = new ArrayList<>();
+
+	private final ResponseCodeEnum[] permissiblePrechecks = new ResponseCodeEnum[] {
+			OK, BUSY, DUPLICATE_TRANSACTION, PLATFORM_TRANSACTION_NOT_CREATED,ACCOUNT_ID_DOES_NOT_EXIST,
+			ACCOUNT_DELETED
+	};
+
+	public static void main(String... args) {
+		parseArgs(args);
+		AccountBalancesClientSaveLoadTest suite = new AccountBalancesClientSaveLoadTest();
+		suite.runSuiteSync();
+	}
+
+	@Override
+	protected List<HapiApiSpec> getSpecsInSuite() {
+		return List.of(new HapiApiSpec[] {	runAccountBalancesClientSaveLoadTest() });
+	}
+
+	@Override
+	public boolean hasInterestingStats() {
+		return false;
+	}
+
+
+	private HapiApiSpec runAccountBalancesClientSaveLoadTest() {
+		PerfTestLoadSettings settings = new PerfTestLoadSettings();
+		return defaultHapiSpec("AccountBalancesClientSaveLoadTest" )
+				.given(
+						tokenOpsEnablement(),
+						withOpContext((spec, ignore) -> settings.setFrom(spec.setup().ciPropertiesMap()))
+
+				).when(
+						sourcing(() -> runWithProvider(accountsCreate(settings))
+								.lasting(() -> totalAccounts / ESTIMATED_CRYPTO_CREATION_RATE + 10,
+										() -> TimeUnit.SECONDS)
+								.totalOpsToSumbit(() ->	(int)Math.ceil(totalAccounts))
+								.maxOpsPerSec(() -> settings.getTps())
+								.maxPendingOps(() -> MAX_PENDING_OPS_FOR_SETUP)),
+
+						sleepFor(20 * SECOND),
```

Comment:
Instead of SECOND we can use Units.SECONDS_TO_MILLISECONDS

Reply:
Right now only `hedera-node` module has dependency on `com.swirlds.com.*`. 'test-clients` module doesn't have such dependency. `com.swirlds.common.Units` class is not visible to `test-clients`. If we want to have that, I guess we need re-org on the pom files of hedera services. But I am not sure if this is a good idea at this moment.

Reply:
Ah got it ! Thanks Leo. We can rename it to SEC_TO_MS

Reply:
That's a good idea. But I guess it would be more valuable to find out the answer for why we don't introduce the `com.swirlds.common.*` to `test-clients` module before we introduce the similar constants in `test-clients` module.